### PR TITLE
Added html for section 1

### PR DESCRIPTION
**Pull request by Tumakoff**

### CODE REVIEW

**Review by Maiia2023 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -10,7 +10,27 @@
...
+            <a class="link" href="https://google.com">
+              <img src="./assets/icons/google.svg" alt="Google-icon">
+            </a>  
+          </div>
+        </div>
+        <div class="image-block">
+          <img src="./assets/images/1.jpg" alt="Bag and device">
+        </div>
+      </div>
+    </section>
```
Comment (nagorapro) [inline index.html]: Use BEM methodology
Use "download" class name
Use "download" id name

**Review by nagorapro — COMMENTED** [review]
Need fix

File: README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#ng-free-zen
```
Comment (KaterinaTuma) [inline README.md:1]: Add spaces between '#', '•' and values
```
# ng-free-zen

• HTML  
• CSS  
• JS
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+•HTML
+•CSS
+•JS
```
Comment (KaterinaTuma) [inline README.md]: Add spaces between '#', '•' and values
```
# ng-free-zen

• HTML  
• CSS  
• JS
```

File: index.html [hunk 4]
```diff
@@ -10,7 +10,27 @@
...
+    <section class="download" id="download">
+      <div class="download__wrapper">
+        <div class="download__text-block">
+          <h1 class="download__title">Peaceful shopping mindful money</h1>
+          <p class="download__copy">
+            See how we can help with making your shopping experience and money management more ZEN.
+          </p>
+          <div class="download__links">
+            <a class="download__link" href="https://apple.com">
+              <img src="./assets/icons/apple.svg" alt="Apple-icon">
```
Comment (KaterinaTuma) [inline index.html]: Change alt="Apple-icon" to alt="Apple icon"

File: index.html [hunk 5]
```diff
@@ -10,7 +10,27 @@
...
+            </a>
+            <a class="download__link" href="https://google.com">
+              <img src="./assets/icons/google.svg" alt="Google-icon">
```
Comment (KaterinaTuma) [inline index.html]: Change alt="Google-icon" to alt="Google icon"

**Review by KaterinaTuma — COMMENTED** [review]
Fix

**Review by KaterinaTuma — APPROVED** [review]
Done

**Pull request merged**